docs/supported-sources/dynamodb.md

```diff
@@ -0,0 +1,40 @@
+# DynamoDB
+
+Amazon [DynamoDB](https://aws.amazon.com/dynamodb/) is a managed NoSQL database service provided by Amazon Web Services (AWS). It supports key-value and document data structures and is designed to handle a wide range of applications requiring scalability and performance. 
+
+## URI format
+
+The URI format for DynamoDB is as follows:
+```plaintext
+dynamodb://dynamodb.<region>.amazonaws.com?access_key_id=<aws_access_key_id>&secret_access_key=<aws_secret_access_key>
+```
+
+URI parameters:
+
+- `access_key_id`: Identifes an IAM account. 
+- `secret_access_key`: Password for the IAM account.
+
+
+## Setting up a DynamoDB integration
+
+### Prerequisites
+* AWS IAM access key pair.
+* A DynamoDB Table that you will to load data from
+
+To obtain the access keys, use the IAM console on AWS. See [IAM Documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for more information.
```

Comment:
are we able to share a bit more on the particular IAM permissions that are needed for this?